packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx

```diff
@@ -39,7 +39,8 @@ export default function OrderBasketItemTile({ orderBasketItem, onItemClick, onRe
             <span className={styles.drugName}>{orderBasketItem.drug?.display}</span>
             <span className={styles.dosageInfo}>
               {' '}
-              &mdash; {orderBasketItem.drug.strength} &mdash; {orderBasketItem.drug.dosageForm.display}
+              &mdash; {orderBasketItem.drug.strength}{' '}
```

Comment:
Please do the check for the drug.strength.

Some of the medications don't have the strength as well.